ts_src/txutils.ts

```diff
@@ -0,0 +1,73 @@
+import { WitnessUtxo } from 'bip174/src/lib/interfaces';
```

Comment:
can we remove this? let's try to be consistent with bitcoinjs and avoid adding unnecessary files.

It is just fine to check that the proofs are not null for a confidential output... so `bufferNotEmptyOrNull` and `isConfidentialUtxo` could be removed... we could add a isConfidential method to the Output type of the transaction module, but let's do this in another PR.

tryToUnblind could be moved directly in the pset module but revert to the precedent version.. trying to unblind one prevout with all the blinding keys is too much time expensive in js.